ccl/changefeedccl: TestParallelIOMetrics failed

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/dd9a96da-df76-4405-99fa-3dd2e6750396?testReportRun=3&testReportShard=5&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [62dcafadd237695fecea81b16f8fab3b73af2125](https://github.com/cockroachdb/cockroach/commits/62dcafadd237695fecea81b16f8fab3b73af2125):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestParallelIOMetrics2619891331
    test_log_scope.go:81: use -show-logs to present logs inline
    changefeed_test.go:9178: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics2619891331
--- FAIL: TestParallelIOMetrics (63.43s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:864: making server as secondary tenant
    helpers_test.go:943: making pubsub feed factory
    changefeed_test.go:9150: condition failed to evaluate within 45s: from changefeed_test.go:9153: waiting for pending keys: 0
    --- FAIL: TestParallelIOMetrics/pubsub (63.43s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=5</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-39369

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/ed4d2532-74d9-42fa-88ed-2331ee14ca92?testReportRun=3&testReportShard=6&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [8551145a0c99c4c95a28ec470e699d0c20ca97ab](https://github.com/cockroachdb/cockroach/commits/8551145a0c99c4c95a28ec470e699d0c20ca97ab):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestParallelIOMetrics2261404791
    test_log_scope.go:81: use -show-logs to present logs inline
    changefeed_test.go:9327: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics2261404791
--- FAIL: TestParallelIOMetrics (140.65s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:872: making server as secondary tenant
    helpers_test.go:951: making pubsub feed factory
    changefeed_test.go:9323: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:9323
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1123
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1158
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestParallelIOMetrics/pubsub
    --- FAIL: TestParallelIOMetrics/pubsub (140.64s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=6</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/dd4879a5-ea10-4f8d-9619-20bbf1925c1e?testReportRun=3&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [dcce4cafa234525fc859d32745c11ed87890dc7b](https://github.com/cockroachdb/cockroach/commits/dcce4cafa234525fc859d32745c11ed87890dc7b):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestParallelIOMetrics745887789
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9409: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics745887789
--- FAIL: TestParallelIOMetrics (203.87s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:883: making server as secondary tenant
    helpers_test.go:962: making pubsub feed factory
    changefeed_test.go:9405: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:9405
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1134
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1169
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestParallelIOMetrics/pubsub
    --- FAIL: TestParallelIOMetrics/pubsub (203.86s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #130390 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2.2-rc release-blocker]
- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/e37017be-7d0b-4aff-a982-cce5aa7ae2b9?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [30dbb173d0f083b35cf9eb8093832a5dd764c5af](https://github.com/cockroachdb/cockroach/commits/30dbb173d0f083b35cf9eb8093832a5dd764c5af):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestParallelIOMetrics2444153207
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9398: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics2444153207
--- FAIL: TestParallelIOMetrics (243.50s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:883: making server as secondary tenant
    helpers_test.go:962: making pubsub feed factory
    changefeed_test.go:9370: condition failed to evaluate within 3m45s: from changefeed_test.go:9373: waiting for pending keys: 0
    --- FAIL: TestParallelIOMetrics/pubsub (243.50s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #130390 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2.2-rc release-blocker]
- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/2c447162-bee7-4e1f-bd1c-1ed11006d63e?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [a0f39e7ac9574756063bc90bba6bc532b45c33d4](https://github.com/cockroachdb/cockroach/commits/a0f39e7ac9574756063bc90bba6bc532b45c33d4):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestParallelIOMetrics2934126764
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9398: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics2934126764
--- FAIL: TestParallelIOMetrics (229.46s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:883: making server as secondary tenant
    helpers_test.go:962: making pubsub feed factory
    changefeed_test.go:9394: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:9394
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1134
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1169
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestParallelIOMetrics/pubsub
    --- FAIL: TestParallelIOMetrics/pubsub (229.45s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #130390 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2.2-rc release-blocker]
- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/9b288445-379a-4ceb-b993-9db079a9e185?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [833dadd212fa4b12b1442ae8e00e85ee80a8cdce](https://github.com/cockroachdb/cockroach/commits/833dadd212fa4b12b1442ae8e00e85ee80a8cdce):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestParallelIOMetrics1998238440
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9398: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics1998238440
--- FAIL: TestParallelIOMetrics (203.79s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:883: making server as secondary tenant
    helpers_test.go:962: making pubsub feed factory
    changefeed_test.go:9394: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:9394
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1134
        	            				github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/helpers_test.go:1169
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestParallelIOMetrics/pubsub
    --- FAIL: TestParallelIOMetrics/pubsub (203.78s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #130390 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc X-duplicate branch-release-24.2.2-rc release-blocker]
- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/f9bdf8d3-9c52-410f-ba6f-1f7deb520a5f?testReportRun=2&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [1e5b3c212b45419c960038718c48a5dd75a111a0](https://github.com/cockroachdb/cockroach/commits/1e5b3c212b45419c960038718c48a5dd75a111a0):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestParallelIOMetrics3725494253
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9396: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics3725494253
--- FAIL: TestParallelIOMetrics (244.22s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:883: making server as secondary tenant
    helpers_test.go:962: making pubsub feed factory
    changefeed_test.go:9368: condition failed to evaluate within 3m45s: from changefeed_test.go:9371: waiting for pending keys: 0
    --- FAIL: TestParallelIOMetrics/pubsub (244.22s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #130390 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc X-duplicate branch-release-24.2.2-rc release-blocker]
- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/a968fd48-b9bc-4e71-a2dd-3e1224ebd2cc?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [f9918d8f81a1829df63ac734fd6d21c60141e338](https://github.com/cockroachdb/cockroach/commits/f9918d8f81a1829df63ac734fd6d21c60141e338):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestParallelIOMetrics382625186
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9396: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics382625186
--- FAIL: TestParallelIOMetrics (239.42s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:883: making server as secondary tenant
    helpers_test.go:962: making pubsub feed factory
    changefeed_test.go:9392: 
        	Error Trace:	pkg/ccl/changefeedccl/changefeed_test.go:9392
        	            				pkg/ccl/changefeedccl/helpers_test.go:1134
        	            				pkg/ccl/changefeedccl/helpers_test.go:1169
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestParallelIOMetrics/pubsub
    --- FAIL: TestParallelIOMetrics/pubsub (239.41s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #133817 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2.5-rc release-blocker]
- #133335 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.1 release-blocker]
- #130390 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc X-duplicate branch-release-24.2.2-rc release-blocker]
- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/changefeedccl.TestParallelIOMetrics [failed](https://tanzanite.cluster.engflow.com/invocations/default/11aee3a6-8a6c-4997-8f59-afecba2f6641?testReportRun=2&testReportShard=9&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on master @ [17535c13cfed95db70cd8dfb1ba6a700686f57b1](https://github.com/cockroachdb/cockroach/commits/17535c13cfed95db70cd8dfb1ba6a700686f57b1):


```
=== RUN   TestParallelIOMetrics
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestParallelIOMetrics3275841666
    test_log_scope.go:76: use -show-logs to present logs inline
    changefeed_test.go:9502: -- test log scope end --
test logs left over in: outputs.zip/logTestParallelIOMetrics3275841666
--- FAIL: TestParallelIOMetrics (140.42s)
=== RUN   TestParallelIOMetrics/pubsub
    helpers_test.go:883: making server as secondary tenant
    helpers_test.go:962: making pubsub feed factory
    changefeed_test.go:9498: 
        	Error Trace:	pkg/ccl/changefeedccl/changefeed_test.go:9498
        	            				pkg/ccl/changefeedccl/helpers_test.go:1134
        	            				pkg/ccl/changefeedccl/helpers_test.go:1169
        	Error:      	Received unexpected error:
        	            	context deadline exceeded
        	Test:       	TestParallelIOMetrics/pubsub
    --- FAIL: TestParallelIOMetrics/pubsub (140.42s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=9</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #133817 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2.5-rc release-blocker]
- #133335 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.1 release-blocker]
- #130390 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc X-duplicate branch-release-24.2.2-rc release-blocker]
- #129917 ccl/changefeedccl: TestParallelIOMetrics failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallelIOMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -9480,6 +9480,10 @@ func TestParallelIOMetrics(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	// This test relies on messing with timings to see pending rows build up,
+	//  so skip it when the system is loaded.
+	skip.UnderDuress(t)
+
 	// Add delay so queuing occurs, which results in the below metrics being
 	// nonzero.
 	defer testingEnableQueuingDelay()()

